<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonValidatePath">
      <property name="text">
       <string>Validate and Init</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditPath">
      <property name="placeholderText">
       <string>Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBoxGitControl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Git Control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGitRemote">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Remote</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonGitFetch">
            <property name="text">
             <string>git fetch --&gt; Clone⏵</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonGitPull">
            <property name="text">
             <string>git pull --&gt; Working Files⏵</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonRemoteAdd">
            <property name="text">
             <string>git remote add origin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditRemoteUrl">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Remote URL</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGitBranches">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Branches</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Clone</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButtonGitReset">
                 <property name="text">
                  <string>git reset --&gt; Branches⏵</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonGitResetHard">
                 <property name="text">
                  <string>git reset --hard --&gt; Working Files⏵</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButtonGitRemote">
                 <property name="text">
                  <string>⏴Remote &lt;-- git remote</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="pushButtonGitCheckout">
                <property name="text">
                 <string>git checkout --&gt; Working Files⏵</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonGitCheckoutTrack">
                <property name="text">
                 <string>⏴Clone &lt;-- git checkout --track</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonGitPush">
              <property name="text">
               <string>⏴Remote &lt;-- git push</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonGitMerge">
              <property name="text">
               <string>git merge --&gt; Working Files⏵</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGitWorkingFiles">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Working Files</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonAddComit">
            <property name="text">
             <string>⏴Branches &lt;-- git commit -a</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditCommitMessage">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Commit message</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonGitStatus">
            <property name="text">
             <string>git status</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
